client/tests/phpunit/Service/Client/Sirius/SiriusApiGatewayClientTest.php

```diff
@@ -0,0 +1,165 @@
+<?php declare(strict_types=1);
+
+
+namespace AppBundle\Service\Client\Sirius;
+
+
+use AppBundle\Service\AWS\RequestSigner;
+use DateTime;
+use DigidepsTests\Helpers\SiriusHelpers;
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Serializer\Serializer;
+
+class SiriusApiGatewayClientTest extends KernelTestCase
```

Comment:
Is this file still needed now we have the other `SiriusApiGatewayClientTest`?

Reply:
Hmmmm I guess we are doubling up a little here. I'll leave it with the get request test in for now. We'll need to update that to Pact too. I'll add a ticket for it.

Reply:
Fair enough 👍